modules/account-map/remote-state.tf

```diff
@@ -1,10 +1,9 @@
 module "accounts" {
   source  = "cloudposse/stack-config/yaml//modules/remote-state"
-  version = "0.19.0"
+  version = "0.22.1"
 
-  component               = "account"
-  privileged              = true
-  stack_config_local_path = "../../../stacks"
```

Comment:
I'm seeing `stack_config_local_path` as required in `0.22.1` -- https://github.com/cloudposse/terraform-yaml-stack-config/tree/0.22.1#input_stack_config_local_path

Have you tested that removing this works?

Reply:
@Gowiem it works without it. there are other instances in the repo where 0.22.1 does not include `stack_config_local_path`

Reply:
Ah -- Apologies @mikedizon, you're correct. I'm missing the fact that this is the `remote-state` child submodule, not the root child module. 

Let me re-review...